fabbwled-backend/src/main/java/ch/bbw/fabbwled/lands/character/PlayerDto.java

```diff
@@ -140,6 +140,14 @@ public boolean hasCodeword(String codeword) {
         return codeWords.contains(codeword);
     }
 
+    public boolean hasProfession(String profession) {
+        return profession().toString().equalsIgnoreCase(profession);
```

Comment:
to string? why not keep it an enum?

Reply:
yes, please use the `ProfessionEnum` here

Reply:
good input, I deleted the method and used the enum for checks